buildUtilities/checkTask.m

```diff
@@ -0,0 +1,7 @@
+function checkTask(~)
+% Identify code issues
+        issues = codeIssues('../toolbox');
```

Comment:
Instead of using relative paths, please call the project object and build the full path according to the root project folder. Example:
`
projObj = currentProject;
fullfile(projObj.RootFolder,'toolbox');
`